fmf.spec

```diff
@@ -1,16 +1,19 @@
 Name: fmf
-Version: 1.3.0
-Release: 1%{?dist}
+Version: 0.0.0
+Release: %autorelease
 
 Summary: Flexible Metadata Format
 License: GPLv2+
 BuildArch: noarch
 
-URL: https://github.com/psss/fmf
-Source0: https://github.com/psss/fmf/releases/download/%{version}/fmf-%{version}.tar.gz
+URL: https://github.com/teemtee/fmf
+Source0: https://github.com/teemtee/fmf/releases/download/%{version}/fmf-%{version}.tar.gz
 
 # Main fmf package requires the Python module
-Requires: python%{python3_pkgversion}-%{name} == %{version}-%{release}
+BuildRequires: python3-devel
+BuildRequires:  python3dist(docutils)
```

Comment:
I don't think it needs docutils.

Reply:
I think that one was because of `rst2man` or something similar.

Reply:
oh I see you're creating the man in rpm build. Interesting. Does it work well?